### DISCUSSION TITLE
Add previous execution time next to elapsed time in cell

### DISCUSSION THREAD

**Discussion by charris-msft** [microsoft/vscode-jupyter#9818] [New Ideas]
Working with large dataframes means that often it can take several minutes to execute a cell.
I have started adding a comment at the bottom of my long-running cells indicating the time it recently took for the cell to execute (orange box).
This has proven to be super helpful so I think it would make a great feature. See the red text at the bottom of the picture.

![image](https://user-images.githubusercontent.com/74415662/165788727-3fefcee9-79cc-424f-9b35-bc156f893e07.png)

**Comment by IanMatthewHuff** [comment]
I like that suggestion personally. If we put this on our backlog we might pull this discussion item into a work item. Probably best as an optional setting though. Might be a bit noisy for the general case.

**Reply by DonJayamanne** [reply]
I think this is a duplicate of [Show time stamp and zone when code cell last ran](https://github.com/microsoft/vscode-jupyter/issues/8245)
@charris-msft please could you upvote that issue

**Reply by charris-msft** [reply]
@DonJayamanne - close enough. I upvoted it and added a comment at the bottom to include the persistence that I'm looking for.

**Reply by charris-msft** [reply]
@IanMatthewHuff - 
you've got a good point about noisy for the general case - you could consider only enabling it automatically for cells that take a long time to run (what ever "long time" means 😊).

And I wouldn't suggest using red text - I just used that in my example to make it easy to see.
I would use something a bit less prominent than the elapsed time size/color.
